powerfactory/powerfactory-converter/src/main/java/com/powsybl/powerfactory/converter/AbstractConverter.java

```diff
@@ -21,6 +21,8 @@
  */
 public abstract class AbstractConverter {
 
+    static final String TYP_ID = "typ_id";
```

Comment:
Maybe we can more all attribute names in a dedicated utility class `DataAttributeNames`?

Reply:
Done